src/pages/2021/sponsorlogistics.mdx

```diff
@@ -2,9 +2,105 @@ import { SiteLayout } from "../../components/layout/SiteLayout";
 
 <SiteLayout title="Schedule">
 
-# Conference Sponsor Logistics
+# 2021 Conference Sponsor Logistics
 
-The oSTEM conference team is hard at work in the planning stages of the 2021
-conference. Check back here later for the full list of sponsors.
+Thank you for sponsoring the oSTEM 2021 Virtual Annual Conference! This page contains the latest information regarding conference planning and sponsor participation. Please be sure to revisit this page to receive the the most up-to-date information. 
+
+# Table of Contents
+
+- [Key Deadlines and Milestones](#deadlines)
+
+- [Conference Schedule and Programming Highlights](#schedule)
+
+- [Sponsored Workshops](#workshops)
+
+    - [50-minute Sponsored Workshop Submission](#submit)
+
+    - [What should you submit?](#submit-ideas)
+
+    - [Are you stuck and don't know what to present?](#present-ideas)
+
+- [Organization Logo Requirements](#)
+
+- [Program Advertisement Requirements](#)
+
+- [Virtual Platforms](#)
+
+- [Career Fair via Brazen](#)
+
+- [Contact Information, Resumes/CVs](#)
+
+- [oSTEM Career Center](#)
+
+- [Awards Information](#)
+
+
+# <a name="deadlines"></a> Key Deadlines and Milestones
+
+## Sponsor Key Deadlines and Milestones
+
+# <a name="schedule"></a> Conference Schedule and Programming Highlights
+
+Here's where you can find the [latest high-level conference schedule.](https://conference.ostem.org/2021/schedule)
+
+**Attendee Types**
+
+- **Full-Conference Student Attendees -** People who are currently full-time or part time students.
+
+- **Full-Conference Professional Attendees -** These attendees may and are encouraged to attend any part of the conference. 
+
+- **Full-Conference Sponsor Attendees -** These attendees are attending the conference not as oSTEM members, but as oSTEM sponsors.
+
+- **Career Expo only Attendees -** These attendees do not need to register via the conference Bizzabo Portal - Only need to register to Brazen. They will receive an email invite from Brazen after the sponsor Booth Owner adds them as Booth Rep in Brazen.
+
+**Pre-Conference Oppertunities:**
```

Comment:
Again, this should be a heading. If I see any more like this, I'm not gonna leave a comment. ;)